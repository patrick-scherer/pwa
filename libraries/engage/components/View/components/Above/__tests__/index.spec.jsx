import React from 'react';
import { shallow } from 'enzyme';
import Above from '../index';

jest.mock('../../../context');

describe('engage > components > view > components > above', () => {
  it('should render above', () => {
    const wrapper = shallow(<Above />).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
