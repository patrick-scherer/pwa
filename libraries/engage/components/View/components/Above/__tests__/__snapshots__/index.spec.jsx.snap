// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`engage > components > view > components > above should render above 1`] = `
<div
  aria-hidden={true}
  style={
    Object {
      "flexShrink": 0,
      "height": "calc(20px + var(--safe-area-inset-top))",
    }
  }
/>
`;
